boards/nucleo-l073/include/periph_conf.h

```diff
@@ -178,7 +178,13 @@ static const spi_conf_t spi_config[] = {
  * @name    ADC configuration
  * @{
  */
-#define ADC_NUMOF           (0)
+#define ADC_CONFIG {             \
+    { GPIO_PIN(PORT_A, 0), 17 }, \
```

Comment:
According to the [nucleo boards datasheet](http://www.st.com/etc/clientlibs/st-site/media/app/images/icon_pdf.png) section 6.11, table 18 page 46, the default configuration of ADC pins should be something like:
```c
/**
 * @name   ADC configuration
 * @{
 */
#define ADC_CONFIG {            \
    { GPIO_PIN(PORT_A, 0), 0 }, \
    { GPIO_PIN(PORT_A, 1), 1 }, \
    { GPIO_PIN(PORT_A, 4), 4 }, \
    { GPIO_PIN(PORT_B, 0), 8 }, \
    { GPIO_PIN(PORT_C, 1), 11 },\
    { GPIO_PIN(PORT_C, 0), 10 } \
}

#define ADC_NUMOF           (6)
/** @} */
```

Reply:
done